content/en/continuous_integration/setup_pipelines/github.md

```diff
@@ -62,6 +62,14 @@ To enable logs, follow these steps:
 
 Immediately after toggling logs collection, workflow job logs are forwarded to Datadog Logs. Note that logs are billed separately from CI Visibility. Log retention, exclusion, and indexes are configured in Logs Settings.
 
+### Infrastructure metric correlation
+
+If you are running the GitHub runners you can correlate your jobs to the host is running. To do so make sure the GitHub runner name
+matches the hostname of the machine it is running on. CI Visibility will automatically detect the runner name and link it to the
```

Comment:
```suggestion
If you are using self-hosted GitHub runners you can correlate jobs to the host that is running them. To do so make sure the GitHub runner name
matches the hostname of the machine it is running on. CI Visibility uses this to link to
```
Not sure if self-hosted is the best word for Github but _you are running the GitHub runners_ sounds a bit too informal to me